workflows/methylation/methylation-cohort.wdl

```diff
@@ -0,0 +1,245 @@
+version 1.1
+
+workflow methylation_cohort {
+    meta {
+        description: "Process methylation data for a cohort of samples"
+        outputs: {
+            combined_beta: "Matrix (in CSV format) containing beta values for every (common) probe on the array as rows and all of the input samples as columns.",
+            filtered_beta: "Matrix (in CSV format) containing only beta values for the retained probes (top N highest standard deviation) for all provided samples.",
+            filtered_probeset: "List of probe names that were retained after filtering to the top N highest standard deviation",
+            umap_embedding: "UMAP embedding for all samples",
+            umap_plot: "UMAP plot for all samples",
+        }
+    }
+
+    parameter_meta {
+        unfiltered_normalized_beta: "Array of unfiltered normalized beta values for each sample"
+        max_length: "Maximum number of beta files to merge before using iteration"
```

Comment:
I wonder if this should be hidden somehow... Maybe `group: "Advanced"` or even `"Developer Options"`?

Reply:
Before going that route, I'd probably just move it to a private declaration. I don't think anyone should change it, but that'd make it much clearer that it probably shouldn't be changed.

Reply:
Having it as a private declaration sounds good to me. Should we make that change? I think we either need to move it so it can't be modified or need to add something to `meta` clarifying users shouldn't touch it.